oci/otel-collector/README.md

```
@@ -0,0 +1,155 @@
+    sdk -->|"OTLP gRPC / HTTP\nover Linkerd mTLS\n(transparent to SDK)"| ep
+    label -.->|"read by k8sattributes\npromoted to span attribute"| enrich
+    ep --> enrich
+    enrich --> sampling
+    enrich --> filter
+    sampling -->|"sampled spans"| ai
+    filter -->|"WARN+ log records"| ai
+    enrich -->|"all metrics"| amw
+```
+
+**SDK endpoint** (same value for gRPC or HTTP):
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)